guides/plugins/plugins/storefront/add-icons.md

```diff
@@ -0,0 +1,85 @@
+# Add custom icons
+
+## Overview
+
+In this guide you will learn how to use the icon renderer component as well as adding custom icons.
+
+## Prerequisites
+
+In order to follow this guide easily, you first need to have a functioning plugin installed. Head over to our 
+[plugin base guide](../plugin-base-guide.md) to create a plugin, if you don't know how it's done yet.
+Also knowing and understanding SCSS will be quite mandatory to fully understand what's going on here.
+Furthermore, it might be helpful to read the guide on how to [handle own assets](./add-custom-assets.md) in your 
+plugin before you start with this one.
+
+## Adding icon
+
+In order to add any icons to the Storefront, you use our `sw_icon` twig action. This way, an icon of choice is 
+displayed in the Storefront.
+
+Needless to say, the first step is saving your image somewhere in your plugin Shopware can find it. The default path 
+for icons is thw following:
+```
+<YourTheme>/src/Resources/app/storefront/dist/assets/icon/default
```

Comment:
Theme? Or rather `<YourPlugin>`?

Reply:
Seems to be out of habit, `Plugin` of course